x/auth/tx/direct_aux.go

```diff
@@ -54,15 +53,12 @@ func (signModeDirectAuxHandler) GetSignBytes(
 
 	feePayer := protoTx.FeePayer()
 
+	feepayer := sdk.AccAddress(feePayer)
```

Comment:
Let’s have a different name instead of having the same variable name with a different casing.